### PR TITLE
Adobe Campaign Classic clean up

### PR DESCRIPTION
**Pull request by calebk1**

### CODE REVIEW

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md [hunk 1]
```diff
@@ -108,36 +108,37 @@ The `registerDevice` API registers a device with your Campaign Classic registrat
...
 
-#### Swift
+### Swift
+
+**Example**
 
 ```swift
 ACPCampaignClassic.registerDevice(deviceToken, userKey: userKey, additionalParams: additionalParams, callback: {(_ success: Bool?) -> Void in
-            NSLog("Registration Status: %d", success)                                                                                                                        })
+            NSLog("Registration Status: %d", success)})
```
Comment (emdobrin) [inline using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md:117]: can you revert this change for code indentation?

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md [hunk 2]
```diff
@@ -2,9 +2,9 @@
 
 ## Events handled
 
-The following events are handled by the Campaign Classic extension:
+The following events are handled by the Adobe Campaign Classic extension:
 
-### Campaign Request Content   <a id="campaignclassic-content-request"></a>
+### Campaign request
```
Comment (emdobrin) [inline using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md:7]: The type of the event is actually Campaign request content, so let's keep it the same. 
Please note that there are multiple types of request events, such as request content, request identity or request reset.

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md [hunk 3]
```diff
@@ -22,13 +22,13 @@ Here are the key-value pairs in this event:
...
-    "registerdevice": <Boolean>
-    "devicetoken": <String>
-    "userkey": <String>
-    "additionalparameters": <VariantMap>
-    "deviceinfo": <StringMap>
+    "registerdevice": "<Boolean>",
+    "devicetoken": "<String>",
+    "userkey": "<String>",
+    "additionalparameters": "<VariantMap>",
+    "deviceinfo": "<StringMap>"
```
Comment (emdobrin) [inline using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md:31]: Adding quotes in a json example may indicate that the value is a string for a developer. Can we revert this change in all similar examples?

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md [hunk 4]
```diff
@@ -51,22 +51,20 @@ The `trackNotificationReceive` parameters are also sent in the event, as describ
...
 {
-    "trackclick": <Boolean>
-    "trackinfo": <StringMap>
+    "trackclick": "<Boolean>",
+    "trackinfo": "<StringMap>"
 }
 ```
 
-### Configuration Response Content   <a id="configuration-response-content"></a>
+### Configuration response
```
Comment (emdobrin) [inline using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md:61]: Configuration response content

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md [hunk 5]
```diff
@@ -51,22 +51,20 @@ The `trackNotificationReceive` parameters are also sent in the event, as describ
...
 
 The data property in this event is used by each extension to modify its settings. Each extension is responsible for reading out the part of the data property for which it is concerned.
 
-For more information on the data payload definition in this event, see the [Configuration Keys](./#configuration-keys) section.
+#### Data payload definition 
 
-#### Data payload definition   <a id="data-payload-definition-2"></a>
-
-The Campaign Classic extension also reads the following key from the configuration event:
+The following key/value pairs are used by the configuration event in the Adobe Campaign Classic extension.
```
Comment (emdobrin) [inline using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md:67]: The following key/value pairs are used by the Adobe Campaign Classic extension from the configuration event.

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md [hunk 6]
```diff
@@ -76,7 +74,7 @@ The Campaign Classic extension also reads the following key from the configurati
 
 The following events are dispatched by the Campaign Classic extension:
 
-### Campaign Response Content
+### Campaign response
```
Comment (emdobrin) [inline using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md:77]: Campaign response content

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md [hunk 7]
```diff
@@ -1,56 +1,58 @@
...
+**Example**
 
 ```objectivec
 NSString *campaignClassicExtensionVersion = [ACPCampaignClassic extensionVersion];
 ```
 
-**Swift**
+### Swift
+
+**Example**
```
Comment (emdobrin) [inline using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md:29]: One nice thing to do would be to reorder Swift first, then Objective-C across all examples. We can probably do that in a separate PR if you want.

File: using-mobile-extensions/adobe-campaignclassic/README.md [hunk 8]
```diff
@@ -15,7 +15,7 @@
 ![Configuring the Campaign Classic extension](../../.gitbook/assets/acc-configure-extension.png)
 
 {% hint style="info" %}
-Trying to find your Campaign Classic registration or tracking endpoint URLs? You can retrieve this information in the Campaign Classic interface in the **Tools** &gt; **Advanced** &gt; **Deployment wizard** menu. The endpoint for push notifications is usually the same as the URL that is used for web forms and surveys.
+You can retrieve your Campaign Classic registration on tracking endpoint URLS in the Campaign Classic interface under the **Tools > Advanced > Deployment wizard** menu. The endpoint for push notifications is usually the same as the URL that is used for web forms and surveys.
```
Comment (rymorale) [inline using-mobile-extensions/adobe-campaignclassic/README.md:18]: `registration on tracking endpoint URLS` should be reverted back to `registration or tracking endpoint URLS`

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md [hunk 9]
```diff
@@ -1,56 +1,58 @@
...
-### Registering a user's device with Campaign Classic
+The `registerDevice` API lets you register a user device with Campaign Classic.
 
 {% tabs %}
 {% tab title="Android" %}
-To prepare your app to handle push notifications, see [Set up a Firebase Cloud Messaging client app on Android](https://firebase.google.com/docs/cloud-messaging/android/client). After you receive the Firebase Cloud Messaging (FCM) SDK registration token, send this token and the device information to Campaign Classic by using the `registerDevice` API.
+### Java
 
-### registerDevice
+To prepare your app to handle push notifications, see the tutorial on [setting up a Firebase Cloud Messaging client app on Android](https://firebase.google.com/docs/cloud-messaging/android/client). After you receive the Firebase Cloud Messaging \(FCM\) SDK registration token, send this token and the device information to Campaign Classic by using the `registerDevice` API.
```
Comment (rymorale) [inline using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md:45]: the forward slashes at `\(FCM\)` can be removed.

**Comment by emdobrin** [comment]
@calebk1 we provided some feedback on this PR, let us know if you are still working on it or when you need the final review.

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md (hunk 3 above)
Comment (calebk1) [reply using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md:31]: Hmm, I know speaking as a person who does do development that I don't think I'd be confused by that?

Let me check what we do on the Platform-side of things, and get back to you on that.

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md [hunk 10]
```diff
@@ -2,71 +2,102 @@
...
-```text
+```json
 {
-    "trackclick": <Boolean>
-    "trackinfo": <StringMap>
+    "trackClick": true,
+    "trackInfo": {
+        "_mId": "12345",
+        "_dId": "f0190"
+    }
```
Comment (swarna04) [inline using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md:90]: 👍 I like the idea of including examples rather than \<type\> info again.

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md [hunk 11]
```diff
@@ -80,21 +82,31 @@ public void onNewToken(String token) {
...
-
-### registerDevice
+{% hint style="info"}
+To get your app ready to handle push notifications, see the tutorial on [configuring remote notification support](https://developer.apple.com/library/archive/documentation/NetworkingInternet/Conceptual/RemoteNotificationsPG/HandlingRemoteNotifications.html#//apple_ref/doc/uid/TP40008194-CH6-SW1). After you receive the Apple Push Notification service (APNs) token, send this token and the device information to Campaign Classic using the `registerDevice` API.
 
 The `registerDevice` API registers a device with your Campaign Classic registration server. It takes the APNS token as a parameter with a user key that identifies a user, such as an email address or a login name. You can also provide a map of the custom key-value pairs that you want to associate with the registration. A boolean value is returned in the callback, which signals whether the registration was successful.
+{% endhint %}
+
+### Swift
```
Comment (swarna04) [inline using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md:91]: Have we done this change in other extension docs too? I find this format a little odd
```
Swift -> Example
Objective-C - > Syntax, Example
```
(1) We should include syntax with Swift too; or 
(2) Keep syntax first (Objective-C only in case of 2.x) and then include Swift/ Objective-C examples. Refer to - https://aep-sdks.gitbook.io/docs/using-mobile-extensions/adobe-target/target-api-reference#displayedlocations

Thoughts?

**Review by swarna04 — CHANGES_REQUESTED** [review]

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md (hunk 11 above)
Comment (calebk1) [reply using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md:91]: Yeah, I should also include the syntax for it - that being said, I didn't have the syntax, which is why I wasn't sure what to add.

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md (hunk 6 above)
Comment (swarna04) [reply using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md:110]: This is not fixed still, use `Campaign response content`

**Review by swarna04 — CHANGES_REQUESTED** [review]

**Comment by emdobrin** [comment]
@calebk1 thank you for all the updates so far, can you please review the final round of feedback from Swarna?

**Comment by calebk1** [comment]
Oooh somehow I didn't notice the last set of comments - I should be able to get to this some time tomorrow!

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md [hunk 12]
```diff
@@ -2,71 +2,102 @@
...
+    },
+    "deviceToken": "d6x4Bu6pxDg:APA91bG9jZb6oQ3-sExUZRbaYP_LjA6POO0NXMaUaxcBQj9GF5ZBGlFo76raIabLziMMt2mLhLafBW9kYqiZ3Nemwjox3Hg6muTPyLeMOuGj24mCitVt73_KML184meqLYXCqhvg53MI",
+    "registerDevice": true,
+    "deviceInfo": {
+        "deviceName": "generic_x86",
+        "deviceBrand": "Android",
+        "deviceManufacturer": "unknown",
+        "deviceUuid": "00000000-3398-7b70-0000-000033987b70"
+    },
+    "userKey": "john.doe@adobe.com"
```
Comment (emdobrin) [inline using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-event-reference.md:48]: john@example.com

File: using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md [hunk 13]
```diff
@@ -270,21 +289,27 @@ public void onResume() {
...
+func userNotificationCenter(_ center: UNUserNotificationCenter, didReceive response: UNNotificationResponse, withCompletionHandler completionHandler: @escaping () -> Void) {
+      guard let userInfo = response.notification.request.content.userInfo as? [String: String] else {
+          return
+      }
+      ACPCampaignClassic.trackNotificationClick(userInfo)
+      completionHandler()
+}
 ```
 
 ### Objective-C
```
Comment (emdobrin) [inline using-mobile-extensions/adobe-campaignclassic/adobe-campaignclassic-api-reference.md:315]: the format for the obj-c example below is a little off L330-331, can we fix that too?
```
[ACPCampaignClassic trackNotificationClick:response.notification.request.content.userInfo];
completionHandler();
```

**Review by emdobrin — APPROVED** [review]
@calebk1 looks good with minor suggestions, feel free to merge once resolved.
